cli/cluster/cluster.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/docker/machine/libmachine"
 	"github.com/docker/machine/libmachine/host"
 	"github.com/docker/machine/libmachine/state"
-	"github.com/kubernetes/minikube/cli/constants"
+	"k8s.io/minikube/cli/constants"
```

Comment:
It looks like these aren't working: the travis build is failing: https://travis-ci.org/kubernetes/minikube/builds/127043819

Maybe we need canonical import paths? https://golang.org/doc/go1.4#canonicalimports